### PR TITLE
MBS-8303 less spam from jsonrpc client

### PR DESCRIPTION
**Pull request by dsvoronin**
cilogger.debug goes to file (no stdout spam)

### CODE REVIEW

File: subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
+import com.avito.logger.Logger
 import com.avito.test.http.MockDispatcher
 import okhttp3.mockwebserver.MockWebServer
-import com.avito.logger.Logger
-import org.junit.jupiter.api.extension.AfterEachCallback
-import org.junit.jupiter.api.extension.BeforeEachCallback
-import org.junit.jupiter.api.extension.ExtensionContext
-import org.junit.jupiter.api.extension.ParameterContext
-import org.junit.jupiter.api.extension.ParameterResolver
+import org.junit.jupiter.api.extension.*
```
Comment (sergioboi) [inline subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt]: Are Import settings broken?

**Review by sergioboi — APPROVED** [review]

File: subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt (hunk 1 above)
Comment (eugene-krivobokov) [reply subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt]: We don't know how to make it in editorconfig.
These settings are in [IDE preferences](https://avito-tech.github.io/avito-android/docs/contributing/codestyle/#%D0%B5%D1%81%D0%BB%D0%B8-%D0%B0%D0%B2%D1%82%D0%BE%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0-%D1%81%D0%BB%D0%BE%D0%BC%D0%B0%D0%BB%D0%B0%D1%81%D1%8C)

File: subprojects/gradle/performance/src/main/kotlin/com/avito/performance/PerformanceCompareAction.kt [hunk 2]
```diff
@@ -31,13 +31,14 @@ open class PerformanceCompareAction(
             }
 
             override fun exception(msg: String, error: Throwable) {
-                logger.debug(msg, error)
+                logger.critical(msg, error)
             }
```
Comment (eugene-krivobokov) [inline subprojects/gradle/performance/src/main/kotlin/com/avito/performance/PerformanceCompareAction.kt]: @warabei14 , cc

Comment (dsvoronin) [reply subprojects/gradle/performance/src/main/kotlin/com/avito/performance/PerformanceCompareAction.kt]: i'll revert this change, had no intention to change logic

File: subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt (hunk 1 above)
Comment (dsvoronin) [reply subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt]: fixed

**Pull request merged**